### DISCUSSION TITLE
xmlsec 1.2.37

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#452] [Announcements]
The [XML Security Library 1.2.37](https://www.aleksey.com/xmlsec/download.html) release includes the following changes:

- Fixed two regressions from 1.2.36 release: [issue #437](https://github.com/lsh123/xmlsec/issues/437) and [issue #449](https://github.com/lsh123/xmlsec/issues/449).

Thanks to @hendrikdonner and @scaro-axway for reporting these issues!

**Comment by JohnD-Axway** [comment]
Does this release include the OAEP MGF digest option?

**Reply by lsh123** [reply]
No, it's too big of a change. This is small release to specifically address two regressions.

**Reply by lsh123** [reply]
Also, I plan 1.3.0 release in the spring and the MGF changes are tied to some of the other ABI breaking changes I already made so would not apply cleanly on 1.2.x branch. Sorry.

**Reply by JohnD-Axway** [reply]
Thanks for the info.
